### ISSUE TITLE
Menu Bar is Redundant 

### ISSUE THREAD

**Issue by tashsmit** [RosmaryFC/Meme-ify-Me-ARHM#38]
I would say keep it for only the meme title, but get rid of the "settings" button since it doesn't do anything.

**Comment by m47bell** [comment]
I agree, it's just taking up space and not serving a purpose. thanks for the great feedback
